apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -161,4 +161,60 @@ describe("Client API", () => {
       expect(response.body).property("message").to.eq("Keys not found");
     });
   });
+
+  it("tries CREATING a client with a token that does not have the proper permissions", () => {
```

Comment:
The it statement should have more clarity on what the expected outcome is.

it("responds with a 403 when creating api keys without the proper permissions")